.vscode/node-starter-app.code-snippets

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
My fault when setting up the repo! Could you add `.vscode` to the `.gitignore` and/or remove this file?

Reply:
Found myself with some files too, so did this: https://github.com/cds-snc/cppd-medical-report/pull/24

Reply:
I'll pull in the change you merged already and remove the files. 

I was adding some snippets for the macros, it's still a bit fiddly so probably doesn't need to be shared.